charts/model-engine/templates/celery_autoscaler_stateful_set.yaml

```diff
@@ -0,0 +1,98 @@
+{{- if .Values.celery_autoscaler.enabled }}
+{{- $env := .Values.context }}
+{{- $tag := .Values.tag }}
+{{- $service_identifier := .Values.serviceIdentifier }}
+{{- $num_shards := ternary 30 3 (eq .Values.context "prod") }}
+{{- range $base_app := tuple "celery-autoscaler-elasticache" "celery-autoscaler-sqs" }}
```

Comment:
it's probably fine to have this range but I think that we'd be able to choose which one based on whether we're having endpoints use redis or sqs at some point